src/api/helpers/get-populated-application/index.test.ts

```diff
@@ -51,15 +50,6 @@ describe('api/helpers/get-populated-application', () => {
     expect(result.declaration.id).toEqual(application.declaration.id);
   });
 
-  it('should return an application with populated buyer country', async () => {
```

Comment:
Why are we removing this test? The main function `getPopulatedApplication` has not changed?